argus.yaml

```
@@ -112,11 +112,18 @@ userInputValidation:
   itemMaxTTL: "24h" 
 
   # bucketFormatRegex helps define the validity of a bucket through a regular expression.
+  # (Optional) default: ^[0-9a-z][0-9a-z-]{1,61}[0-9a-z]$
   bucketFormatRegex: "^[0-9a-z][0-9a-z-]{1,61}[0-9a-z]$"
 
   # ownerFormatRegex helps define the validity of a bucket through a regular expression.
+  # (Optional) default: ^.{10,60}$
   ownerFormatRegex: "^.{10,60}$"
 
+  # itemDataMaxDepth is the max allowed depth of the Item JSON data field.
+  # If your DB supports up to N nested objects, you should validate the item's 
+  # data field has depth <= N-1.
+  # (Optional) default: 30
+  itemDataMaxDepth: 30
```

MaxDepth implies that the maximum allowed depth would be equal to this value, but the explanation is telling me that it must be less than this value?

---

I think this is what confused me; here is my suggested change:
```suggestion
  # itemDataMaxDepth is the max allowed depth of the Item JSON data field.
  # If your DB supports up to N nested objects, itemDataMaxDepth should be set to 
  # N-1.
  # (Optional) default: 30
  itemDataMaxDepth: 30
```